Hospital.Application.Contracts/IAnalyticsService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Hospital.Application.Contracts.Patients;
+using Hospital.Application.Contracts.Doctors;
+using Hospital.Application.Contracts.Appointments;
+
+namespace Hospital.Application.Contracts;
+
+/// <summary>
+/// Provides analytical operations based on hospital data.
+/// Returns aggregated and filtered data for reporting purposes.
+/// </summary>
+public interface IAnalyticsService
+{
+    /// <summary>
+    /// Retrieves all doctors with at least 10 years of experience.
+    /// </summary>
+    /// <returns>A collection of <see cref="DoctorDto"/> objects.</returns>
+    IEnumerable<DoctorDto> GetDoctorsWith10Plus();
```

Comment:
Добавить модификатор доступа тут и во всех других интерфейсах.
```suggestion
    public IEnumerable<DoctorDto> GetDoctorsWith10Plus();
```